lib/MetricsConsumer.js

```diff
@@ -22,6 +18,8 @@ class MetricsConsumer {
      * @param {number} rConfig.port - redis port
      * @param {object} mConfig - metrics configurations
      * @param {string} mConfig.topic - metrics topic name
+     * @param {number} mConfig.interval - metrics interval for redis keys
```

Comment:
Can you mention the unit of these values? Or even as a suffix to the variable name like `intervalS` is quite common.